src/asmcnc/apps/wifi_app/screen_wifi.py

```diff
@@ -139,7 +144,7 @@
                     height: dp(100)
                     width: dp(220)
                     orientation: "vertical"
-                    padding: [10,0,20,20]   
+                    padding: [10,0,20,-20]   

```

Comment:
Presumably this doesn't cause the text to overflow the box?

Reply:
No, it was overflowing after I added the toggle button so I updated the padding to fix that